doc/nrf/app_dev/ap_protect/index.rst

```diff
@@ -118,19 +118,20 @@ Based on the available implementation types, you can configure the access port p
 
    * - AP-Protect state
      - Related Kconfig option in the |NCS|
-     - Description
+     - Description of the AP-Protect state
      - AP-Protect implementation type
    * - Locked
      - :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_LOCK` for Secure AP-Protect)
-     - In this state, CPU writes to enable and lock AP-Protect. UICR is not modified.
+     - In this state, CPU uses the MDK system start-up file to write to enable and lock AP-Protect. UICR is not modified.
      - Hardware and software
    * - Authenticated
      - :kconfig:option:`CONFIG_NRF_APPROTECT_USER_HANDLING` (:kconfig:option:`CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING` for Secure AP-Protect)
-     - In this state, AP-Protect is left enabled and it is up to the user handler to unlock the device if needed.
+     - In this state, AP-Protect is left enabled and it is up to the user-space code to handle unlocking the device if needed.
+       The MDK will close the application subsystem access port AHB-AP, but not lock it, so the AHB-AP can be reopened via UART or CTRL-AP.
```

Comment:
```suggestion
       The MDK will close the application subsystem access port AHB-AP, but not lock it, so the AHB-AP can be reopened by the fw. 
       Reopening the AHB-AP would typically be preceded by a handshake operation over UART, CTRL-AP Mailboxes or some other communication channel.
```

The FW reopens the access port, not the CTRL-AP / UART directly. The FW should receive and verify the reopening request through some communication channel like UART, CTRL-AP Mailboxes (available on nrf53 only (nrf91 maybe?)) and make sure the request is valid by requiring a cryptographic handshake of some kind, usually asking the requester to decrypt a random piece of information and pass it back for verification.